src/UI/Components/Diagram/actions.ts

```diff
@@ -434,7 +393,13 @@ export function appendEmbeddedEntity(
         }
       }
     });
-    return [instanceAsTable];
+    return instanceAsTable;
+  }
+
+  if (Array.isArray(entityAttributes)) {
```

Comment:
How come this can be something else than an array at this point? Genuine question.

Reply:
`entityAttributes` can be just one standalone object, `rw_meta` vs `rw_files`, so attributes passed to that function can be both and that's why we differentiate

Reply:
Can you add this piece of information in the docstrings where the argument is defined?

Reply:
the type of the attribute is telling the same piece of the information. Do you meant to add an example?

Reply:
No I mean the info about the rw_meta and rw_files 🤔 unless it's already explained in the initial typing

Reply:
I wouldn't put some information about attributes from some specific entity as they can change in the future and we will have some outdated information. I think that it's quite well documented already, as `InstanceAttributeModel` is type for actual attributes, and in that function typing says that it's either one specific object, or array of objects

Reply:
yeah, okay, but that info you are referring to seems to be part of some logic, which might be worth keeping 🤔

Reply:
But isn't it just clear from the function body itself? the internal function deals with the logic for single embedded entity, and after that function we call it, either in map if those embedded instances are in array, or directly for one single time when it's single embedded instance. And the information that the embedded instance can be as single object, but also in the array is not knowledge specific to the instanceComposer

Reply:
I didn't mean add it here... I meant the little piece of information about what is rw_files and what is rw_meta... You know what it is, I don't. This for me a hint to add it somewhere. Not perse here, just somewhere...

Reply:
Oh, okay, I thought you meant here. Anyway still don't think like we should explain attributes from test env services into the frontend docs, primarily because I now mainly the type they have, but that's the information gathered from watching types and looking on service model, but they can change so it would be pointless to track them